i18n/create_json.js

```diff
@@ -79,43 +80,77 @@ const parseStringFile = function (jsonstream) {
  * substitute variables with corresponding data 
  * @param {String} variable 
  * @param {Object} data 
+ * @param {Object} defdata
+ * @return String
  */
-function substituteVariableData(variable, data) {
+const substituteVariableData = (variable, data, defdata) => {
   let result = variable;
   variable.split('${').forEach(split => {
     const len = split.indexOf('}');
     if (len > -1) {
       const var_name = split.substr(0, len);
-      result = result.replace(`$\{${var_name}}`, data[var_name]);
+      if (data[var_name]) {
+        result = result.replace(`$\{${var_name}}`, data[var_name]);
+      } else {
+        result = result.replace(`$\{${var_name}}`, defdata[var_name]);
+      }
```

Comment:
Use fallback language value (en_GB), in case the variable does not exist in the current lang.